homeassistant/components/fronius/config_flow.py

```diff
@@ -16,6 +17,9 @@
 
 from .const import DOMAIN, FroniusConfigEntryData
 
+if TYPE_CHECKING:
+    from homeassistant.components.dhcp import DhcpServiceInfo
+
```

Comment:
Why is `HomeAssistant` always imported but not `DhcpServiceInfo`?
I am not sure what is the best code style is, but I think it should be consistent.

Reply:
I figured the integration would not run without `HomeAssistant` loaded, but doesn't necessary need the `dhcp` integration (could be disabled etc.) but I'm not even sure if this makes any difference.

Have removed `TYPE_CHECKING` now.